source/includes/usage-examples/connect-sample-app.c

```diff
@@ -0,0 +1,31 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main(void) {
+
+  mongoc_database_t *database
+  bson_t *ping, reply;
```

Comment:
```suggestion
  mongoc_database_t *database = NULL;
  bson_t *ping = NULL, reply = BSON_INITIALIZER;
```

Add missing semicolon. Suggest initializing values so `goto cleanup;` is safe to call anywhere after.

Example:

```c
bson_t *ping;

if (something_may_fail()) {
   goto cleanup; // Destroys uninitialized `ping`!
}

ping = BCON_NEW ("ping", BCON_INT32(1));

cleanup:
   bson_destroy (ping);
```